__tests__/app.test.js

```diff
@@ -285,6 +287,30 @@ describe("DELETE /api/comments/:comment_id", () => {
         expect(response.body.msg).toBe("Bad Request");
       });
   });
+});
 
+describe("GET /api/users", () => {
+  test("200: return an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length).toBe(4);
+        const { users } = body;
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
 
-})
+  test("404: returns 'not found' when api users endpoint is invalid due to typo or different endpoint received", () => {
+    return request(app)
+      .get("/api/user")
+      .expect(404)
+      .then(({ body }) => expect(body.msg).toBe("Not Found"));
+  });
```

Comment:
If you have tested an endpoint that doesn't match before, for an incorrect spelling for example, then you don't need to do it again or for every endpoint. Also because the endpoint is mispelled then it would not live within this describe block as it would not go through /api/users. You can test this once, in it's own describe block